services/core-api/app/api/projects/project_summary/models/project_summary.py

```diff
@@ -166,6 +166,11 @@ def project_summary_lead_party_guid(self):
         if self.project.project_lead_party_guid:
             return self.project.project_lead_party_guid
         return None
+    
+    @hybrid_property
+    def project_lead_email(self):
+        project_lead: Party = Party.find_by_party_guid(self.project_summary_lead_party_guid)
```

Comment:
```suggestion
        project_lead = Party.find_by_party_guid(self.project_summary_lead_party_guid)
```
don't need to type the local variable as it's being set right away, it should be infered by the `Party.find_by_party_guid` method's return type, so make the change there and let it infer. 

typing that would be helpful if you needed to initialize it to null then reassign it later.